src/__tests__/App.spec.js

```diff
@@ -59,7 +59,7 @@ describe('initial state', () => {
     );
 
     expect(
-      screen.getByRole('button', { name: /Back to search results/ })
+      screen.getByRole('button', { name: /Go to search home page/ })
```

Comment:
Shouldn't this be the detail view that's rendered, which would have `Back to search results` in it?

Reply:
@flacoman91 want to fill in here?

Reply:
Yes, ignore this.  The way this works is a bit problematic and is being worked on here:
https://github.com/cfpb/ccdb5-ui/pull/436